pluto-update/attic/subscriber.rb

```diff
@@ -2,6 +2,6 @@
 
   def update_subscriptions( config, opts={} )
     # !!!! -- depreciated API - remove - do NOT use anymore
```

Comment:
perhaps should also call Deprecate here?
https://ruby-doc.org/core-3.1.2/Gem/Deprecate.html

Reply:
thanks for the reminder / ping.  give me another week to try to get back to update.